src/Microsoft.Security.Utilities.Core/HighConfidenceMicrosoftSecurityModels/SEC101_166_AzureSearchIdentifiableQueryKey.cs

```diff
@@ -1,28 +1,28 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System.Collections.Generic;
-
-namespace Microsoft.Security.Utilities
-{
-    internal class AzureSearchIdentifiableQueryKey : IdentifiableKey
-    {
-        public AzureSearchIdentifiableQueryKey()
-        {
-            Id = "SEC101/166";
-            Name = nameof(AzureSearchIdentifiableQueryKey);
-        }
-
-        override public string Signature => IdentifiableMetadata.AzureSearchSignature;
-
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Security.Utilities
+{
+    public class AzureSearchIdentifiableQueryKey : IdentifiableKey
+    {
+        public AzureSearchIdentifiableQueryKey()
+        {
+            Id = "SEC101/166";
+            Name = nameof(AzureSearchIdentifiableQueryKey);
+        }
+
+        override public string Signature => IdentifiableMetadata.AzureSearchSignature;
+
         override public IEnumerable<ulong> ChecksumSeeds => new[] { IdentifiableMetadata.AzureSearchQueryKeyChecksumSeed };
 
-        public override string Pattern
-        {
-            get => @$"{WellKnownRegexPatterns.PrefixAllBase64}(?<refine>[{WellKnownRegexPatterns.Base64}]{{42}}{Signature}[A-D][{WellKnownRegexPatterns.Base64}]{{5}}){WellKnownRegexPatterns.SuffixAllBase64}";
-            protected set => base.Pattern = value;
+        public override string Pattern
+        {
+            get => @$"{WellKnownRegexPatterns.PrefixAllBase64}(?<refine>[{WellKnownRegexPatterns.Base64}]{{42}}{Signature}[A-D][{WellKnownRegexPatterns.Base64}]{{5}}){WellKnownRegexPatterns.SuffixAllBase64}";
```

Comment:
>[A-D] [](http://example.com/codeflow?start=122&length=5)

Suvam, here's an example of what we've been discussing. `A-D` here effectively means that the first four bits of this encoded character were required to be `0` to retain the fixed signature while allowing the checksum bytes to be persisted to the final four chars of the decoded key. In retrospect, we probably should have preferred to simply express all data in the encoded key.